pages/linux/pw-cli.md

```diff
@@ -0,0 +1,16 @@
+# pw-cli
+
+> The PipeWire Command Line Interface.
+> More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.
+
+- List all nodes (sinks and sources) along with their IDs:
+
+`pw-cli list-objects Node`
```

Comment:
What is `Node` here? Is it a placeholder or not? I don't see any info about `Node` [here][manned]. I expected smth like `Node` or `list-objects [arguments...]`.

[manned]: https://manned.org/pw-cli.1#head6

Reply:
It's a keyword. A type. Possible values are `pw-cli list-objects Node` and `pw-cli list-objects Device`.

Reply:
What about this change:

```suggestion
`pw-cli list-objects {{Node|Device}}`
```

? You can also modify your description as: `List all nodes/devices (sinks and sources) along with their IDs:`.

Reply:
It'd be `nodes (sinks and sources)/devices`, sources and sinks are types of node. 

After taking a better look at it, possible values are anything from https://docs.pipewire.org/page_objects_design.html. Considering it's too much to list, I'd stick with the original text. Users most likely want to operate on nodes.